flask_jwt_extended/config.py

```diff
@@ -236,6 +236,8 @@ def identity_claim_key(self):
     def user_claims_key(self):
         return current_app.config['JWT_USER_CLAIMS']
 
-config = _Config()
-
+    @property
+    def exempt_methods(self):
+        return set(["OPTIONS"])
```

Comment:
We don't need to support python2.6, so you could write this as the set literal (ex: `{'foo'}`) instead.